### DISCUSSION TITLE
충돌 처리 레이어가 유니티 에디터에서 제대로 처리되지 않는 문제

### DISCUSSION THREAD

**Discussion by HenryAtBraveTurtles** [naverz/zepeto-studio-kor#545] [Issue Report (Legacy)]
안녕하세요.
제페토 게임을 만들다 충돌 처리 레이어가 유니티 에디터에서 제대로 처리되지 않는 문제를 발견하여 버그 리포트합니다.

<img width="427" alt="image" src="https://user-images.githubusercontent.com/100323867/160536890-66c2fa47-512f-4792-a605-a8dfadb7e839.png">
스크린샷과 같이 ZepetoPlayers 컴포넌트에서 Player의 Layer를 지정해주면 해당 Layer에 대해서만 충돌 처리가 되는 것을 의도한 듯한데요.
유니티 플레이모드에서는 저렇게 설정해도 Player캐릭터가 Default Layer와도 충돌 처리가 됩니다.

QR코드나, 제페토 월드를 업로드해서 제페토 앱에서 실행하면 정상적으로 지정된 Player의 Layer만 충돌 처리됩니다.

ZepetoPlayers 컴포넌트에서 Player의 Layer 속성의 의도가 정확히 어떤 것인지 모르겠으나 유니티 에디터와 제페토 앱에서와의 동장 방식이 달라 리포트합니다.

Unity 2020.3.9f1 / ZEPETO.World 1.1.15

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 사용하신 Environment 레이어가 20~27번 사이인지 확인 부탁 드립니다.
https://studio.zepeto.me/kr/guides/tag-and-layer
에디터에서 레이어 충돌이 의도와 다르게 동작하는 부분은 확인해 보겠습니다.

**Reply by Zepeto-TechEvangelist** [reply]
예시로 Player에게 Player레이어를 세팅하고, 큐브에 Enemy 레이어를 설정한 후 충돌 on/off 각각 테스트했을 때는 정상 동작 하였습니다.

<img width="340" alt="스크린샷 2022-03-30 오후 3 36 55" src="https://user-images.githubusercontent.com/100118650/160767191-d7c9448c-5552-4875-9e1b-f55838d18981.png">

<img width="340" alt="스크린샷 2022-03-30 오후 3 37 27" src="https://user-images.githubusercontent.com/100118650/160767197-025b0fac-5800-49d5-9f9c-889718c389b9.png">

**Reply by HenryAtBraveTurtles** [reply]
아 제가 16번으로 해서 발생한 것이었네요. 답변 감사합니다!
